committees/views.py

```diff
@@ -52,10 +54,12 @@ class CommitteeListView(ListView):
 
     def get_context_data(self, **kwargs):
         context = super(CommitteeListView, self).get_context_data(**kwargs)
-        context["topics"] = Topic.objects.summary()[:self.INITIAL_TOPICS]
-        context["topics_more"] = Topic.objects.summary().count() > self.INITIAL_TOPICS
         context['tags_cloud'] = Tag.objects.cloud_for_model(CommitteeMeeting)
-        context["INITIAL_TOPICS"] = self.INITIAL_TOPICS
+        if waffle.flag_is_active(self.request, 'show_committee_topics'):
+            context["topics"] = Topic.objects.summary()[:self.INITIAL_TOPICS]
```

Comment:
I we are already here.. perhaps move all the topic gathering to a specific method.. so get_context_data would do one thing